libs/facilities/frontend/view/src/lib/components/issues/issues.page.ts

```diff
@@ -0,0 +1,20 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import { RouterLink } from '@angular/router';
+import { IxModule } from '@siemens/ix-angular';
+
+import { XdBrowsePage } from '../browse';
+import { facilities } from '../facility';
+
+@Component({
+	selector: 'lib-issues',
+	standalone: true,
+	imports: [CommonModule, IxModule, RouterLink, XdBrowsePage],
```

Comment:
format

Reply:
I now ran  "nx run-many -t lint --fix" so all things should be formatted correctly